plugins/janus_videoroom.c

```diff
@@ -1916,6 +1948,7 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 		packet.timestamp = ntohl(packet.data->timestamp);
 		packet.seq_number = ntohs(packet.data->seq_number);
 		/* Go */
+		janus_mutex_lock(&participant->subscribers_mutex);
```

Comment:
Although this might be more of an issue when there are many viewers for a specific publisher, we'll make this test ourselves later on.